### DISCUSSION TITLE
Silence the CoolingFans

### DISCUSSION THREAD

**Discussion by Justdigit** [OpenNeptune3D/OpenNept4une#96] [Q&A]
I have updated the firmware. And i my last printer.cfg i had silenced the fans when doing some minor moves.
There was a member who had modded the printer.cfg. But its no working anymore bacause of PA8 pins (not allowed to be double)

Does somebody how to implent this???

Thanks

**Comment by halfmanbear** [comment]
Replied also in the other discussion
------

The issue isn't the host chip or the MCU overheating, it's the stepper motor drivers on the mainboard (no temperature monitoring).

Unfortunately the hotend heatbreak cooling fan and the mainboard fan are wired in parallel.

With Elegoos configuration these stepper drivers don't get any cooling during moves where the hotend isn't being heated. (Full bed meshing, homing / tramming etc) therefore I set the mainboard fan to come on when the stepper motors are activated. I've configured a 15 min cool-off timer, although, you can most likely reduce this value significantly without issue although I need to do some more testing.

**Reply by wjflier** [reply]
On the original Neptune 4 Max firmware the heatbreak fan didn't come on until it was turned on and went off at a certain temp. maybe 45-50c.
